packit_service/worker/jobs.py

```diff
@@ -134,9 +134,11 @@ def process_comment_jobs(
         self, event: Union[PullRequestCommentEvent, IssueCommentEvent]
     ) -> HandlerResults:
         # packit_command can be `/packit propose-update`
-        msg = f"comment '{event.comment[:35]}'"
+        msg = f"comment '{event.comment[:100]}'"
         try:
-            (packit_mark, *packit_command) = event.comment.split(maxsplit=3)
+            cmd_start_index = event.comment.find(REQUESTED_PULL_REQUEST_COMMENT) 
+            (packit_mark, *packit_command) = event.comment[cmd_start_index:].split(maxsplit=3)
```

Comment:
this is probably good enough for now, but once the commands start accept options and arguments, we'd need to improve heuristics